File: VB/MapOverlaysSample/Form1.vb
Imports DevExpress.XtraMap
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Namespace MapOverlaysSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        #Region "#Overlays"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            Dim overlayWithText As MapOverlay = New MapOverlay With {.Alignment = ContentAlignment.BottomRight, .JoiningOrientation = Orientation.Vertical, .Margin = New Padding(0, 4, 8, 8), .Padding = New Padding(7)}
            overlayWithText.Items.Add(New MapOverlayTextItem With {.Text = "Copyright © 2015. Microsoft and its suppliers. All rights reserved."})
            map.Overlays.Add(overlayWithText)

            Dim baseUri As New Uri(System.Reflection.Assembly.GetEntryAssembly().Location)
            Dim overlayWithImage As MapOverlay = New MapOverlay With {.Alignment = ContentAlignment.BottomRight, .JoiningOrientation = Orientation.Vertical, .Margin = New Padding(0, 0, 8, 4), .Padding = New Padding(0)}
            overlayWithImage.Items.Add(New MapOverlayImageItem With {.ImageUri = New Uri(baseUri, "..\..\Images\BingLogo.png")})
            map.Overlays.Add(overlayWithImage)
        End Sub
        #End Region ' #Overlays
    End Class
End Namespace

File: CS/MapOverlaysSample/Form1.cs
using DevExpress.XtraMap;
using System;
using System.Drawing;
using System.Windows.Forms;

namespace MapOverlaysSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        #region #Overlays
        private void Form1_Load(object sender, EventArgs e) {
            MapOverlay overlayWithText = new MapOverlay {
                Alignment = ContentAlignment.BottomRight,
                JoiningOrientation = Orientation.Vertical,
                Margin = new Padding(0, 4, 8, 8),
                Padding = new Padding(7)
            };
            overlayWithText.Items.Add(new MapOverlayTextItem {
                Text = "Copyright © 2015. Microsoft and its suppliers. All rights reserved."
            });
            map.Overlays.Add(overlayWithText);

            Uri baseUri = new Uri(System.Reflection.Assembly.GetEntryAssembly().Location);
            MapOverlay overlayWithImage = new MapOverlay {
                Alignment = ContentAlignment.BottomRight,
                JoiningOrientation = Orientation.Vertical,
                Margin = new Padding(0, 0, 8, 4),
                Padding = new Padding(0),
            };
            overlayWithImage.Items.Add(new MapOverlayImageItem { ImageUri = new Uri(baseUri, "..\\..\\Images\\BingLogo.png") });
            map.Overlays.Add(overlayWithImage);
        }
        #endregion #Overlays
    }
}
